Erro na linha 7: a função 'inner_func' não possui a instrução 'retorna'.
